README.md

```diff
@@ -1,9 +1,84 @@
-# Splunk> Phantom
+[comment]: # "Auto-generated SOAR connector documentation"
+# Discord
 
-Welcome to the open-source repository for Splunk> Phantom's discord App.
+Publisher: discord  
+Connector Version: 1.0.0  
+Product Vendor: Discord  
+Product Name: Discord  
+Product Version Supported (regex): ".\*"  
+Minimum Product Version: 6.2.2.134  
 
-Please have a look at our [Contributing Guide](https://github.com/Splunk-SOAR-Apps/.github/blob/main/.github/CONTRIBUTING.md) if you are interested in contributing, raising issues, or learning more about open-source Phantom apps.
+Integrate with Discord to post messages and attachments to channels
 
-## Legal and License
+### Configuration Variables
+The below configuration variables are required for this Connector to operate.  These variables are specified when configuring a Discord asset in SOAR.
 
-This Phantom App is licensed under the Apache 2.0 license. Please see our [Contributing Guide](https://github.com/Splunk-SOAR-Apps/.github/blob/main/.github/CONTRIBUTING.md#legal-notice) for further details.
+VARIABLE | REQUIRED | TYPE | DESCRIPTION
+-------- | -------- | ---- | -----------
+**token** |  required  | password | Discord bot token
```

Comment:
Some URL to the discord support page? Maybe: https://support-dev.discord.com/hc/en-us/articles/6470840524311-Why-can-t-I-copy-my-bot-s-token